tapeout_16/scripts/layout_Tile.tcl

```diff
@@ -72,7 +74,16 @@ createRouteBlk -name cut01M1 -layer M1 -cutLayer all -box [list 0 [expr $height
 createRouteBlk -name cut02M1 -layer M1 -cutLayer all -box [list 0 -1 $width 0.5]
 
 if $::env(PWR_AWARE) {
-   modifyPowerDomainAttr AON -box 49.95 [expr $height - 9.792] 64.98 [expr $height - 1.152]  -minGaps 0.576 0.576 0.18 0.18
+   set pp_x 0.09
+   set pp_y 0.576
+   set aon_width 14
+   set aon_height 10
```

Comment:
How are you calculating the size of the region? I'm alright with leaving this for now, but ideally we could calculate this from the area of the cell inside the region.